Makefile

```diff
@@ -1226,7 +1226,7 @@ lint: bin/returncheck bin/roachvet bin/optfmt
 	@# Run 'go build -i' to ensure we have compiled object files available for all
```

Comment:
> The go build and go test commands no longer accept the -i flag, which has been [deprecated since Go 1.16](https://go.dev/issue/41696).

https://tip.golang.org/doc/go1.20